### PR TITLE
docs: document retry settings in sample

### PR DESCRIPTION
**Pull request by olavloite**
Fixes https://github.com/GoogleCloudPlatform/java-docs-samples/issues/5310

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/spanner/CustomTimeoutAndRetrySettingsExample.java [hunk 1]
```diff
@@ -53,14 +53,23 @@ static void executeSqlWithCustomTimeoutAndRetrySettings(
...
+                .setMaxRetryDelay(Duration.ofSeconds(10))
+                // The previous wait time is multiplied by this multiplier to come up with the next
+                // wait time, until the max is reached.
                 .setRetryDelayMultiplier(1.5)
 
                 // Configure RPC and total timeout settings.
-                .setInitialRpcTimeout(Duration.ofSeconds(60))
-                .setMaxRpcTimeout(Duration.ofSeconds(60))
-                .setRpcTimeoutMultiplier(1.0)
+                // Timeout for the first RPC call. Subsequent retries will be based off this value.
```
Comment (hengfengli) [inline samples/snippets/src/main/java/com/example/spanner/CustomTimeoutAndRetrySettingsExample.java:66]: In the [doc](https://cloud.google.com/spanner/docs/custom-timeout-and-retry#java): 

> In the following sample, a timeout of 60 seconds is set for the given operation. If the operation takes longer than this timeout, a DEADLINE_EXCEEDED error is returned.

If we update the code sample to this, I don't it match this statement. We probably should update nodejs code sample to set all values to 60s.

Comment (olavloite) [reply samples/snippets/src/main/java/com/example/spanner/CustomTimeoutAndRetrySettingsExample.java:66]: Oh, good catch. I'll change that back, and I'll also open a PR for the Node sample to be in line with the documentation page.

**Review by hengfengli — APPROVED** [review]
LGTM.

**Comment by olavloite** [comment]
The Windows build error is addressed by #1218

**Pull request merged**